DashAI/back/dataloaders/classes/dashai_dataset.py

```diff
@@ -217,6 +226,169 @@ def save_dataset(datasetdict: DatasetDict, path: str) -> None:
         )
 
 
+@beartype
+def check_split_values(
+    train_size: float,
+    test_size: float,
+    val_size: float,
+) -> None:
+    if train_size < 0 or train_size > 1:
+        raise ValueError(
+            "train_size should be in the (0, 1) range "
+            f"(0 and 1 not included), got {val_size}"
+        )
+    if test_size < 0 or test_size > 1:
+        raise ValueError(
+            "test_size should be in the (0, 1) range "
+            f"(0 and 1 not included), got {val_size}"
+        )
+    if val_size < 0 or val_size > 1:
+        raise ValueError(
+            "val_size should be in the (0, 1) range "
+            f"(0 and 1 not included), got {val_size}"
+        )
+
+
+@beartype
+def split_indices(
```

Comment:
Prefer indexes over indices

```suggestion
def split_indexes(
```

Reply:
Same in all function params names and params.